src/screens/WalletMigrationArchanova/Review/MigrationSummary.js

```diff
@@ -124,7 +124,7 @@ function MigrationSummary({ etherspotAccount, archanovaAccount, items, feeInEth,
 
       {feeInEth && highFee && (
         <WarningBlock
-          text={tRoot('transactions.highGasFee.warningLabel')}
+          text={tRoot('transactions.highGasFee.ethereumNetworkFees')}
```

Comment:
Same as above: https://github.com/pillarwallet/pillarwallet/pull/3042

Reply:
Okay, I'll revert it.